### DISCUSSION TITLE
Visual Mode Actions closes

### DISCUSSION THREAD

**Discussion by sohanemon** [olimorris/codecompanion.nvim#269] [Q&A] [answered]
I recently transitioned from using Telescope to fzf-lua and encountered an issue that I would like assistance with. I’ve recorded a video demonstrating the problem, which you can find here: [issue video](https://github.com/user-attachments/assets/723d413b-3181-48b1-adf3-75ba3543dc8b)

Here it closes the action menu in visual mode, though normal mode is working perfectly. For your reference, I am currently using LazyVim on Arch Linux.

Any guidance or suggestions would be greatly appreciated!

Thank you.

**Comment by olimorris** [comment]
Hey @sohanemon, have you you tried this out recently? Is it still an issue?

**Reply by bassamsdata** [reply, marked as answer]
@sohanemon This was indeed an issue a week ago, but it should now work perfectly. That said, I've never tried fzf-lua before, I'll give it a shot and see how it works with visual selection.

**Reply by bassamsdata** [reply]
I've tried Fzf-lua with LazyVim distro (updated everything to the latest version) and it worked fine. if still having an issue, would you mind sharing the nvim version as well?
![Screen Recording 2024-12-10 at 10 33 11 AM](https://github.com/user-attachments/assets/2dfa872e-b0e2-4df7-b642-325367a6b949)

**Reply by sohanemon** [reply]
> Hey @sohanemon, have you you tried this out recently? Is it still an issue?

I have enabled just now. And its working perfectly.
